--- a/kernel/rh_messages.c	2022-02-21 14:17:46.000000000 +0000
+++ b/kernel/rh_messages.c	2022-04-04 20:32:36.906594860 +0000
@@ -3,15 +3,15 @@
 
 #define DEV_DESC_LEN 256
 /*
- * The following functions are used by Red Hat to indicate to users that
- * hardware and drivers are unsupported, or have limited support in RHEL major
+ * The following functions are used by EuroLinux to indicate to users that
+ * hardware and drivers are unsupported, or have limited support in EuroLinux major
  * and minor releases.  These functions output loud warning messages to the end
  * user and should be USED WITH CAUTION.
  *
- * Any use of these functions _MUST_ be documented in the RHEL Release Notes,
+ * Any use of these functions _MUST_ be documented in the EuroLinux Release Notes,
  * and have approval of management.
  *
- * Generally, the process of disabling a driver or device in RHEL requires the
+ * Generally, the process of disabling a driver or device in EuroLinux requires the
  * driver or device to be marked as 'deprecated' in all existing releases, and
  * then either 'unmaintained' or 'disabled' in a future release.
  *
@@ -27,7 +27,7 @@
  *
  * Called to notify users that the device will no longer be tested on a routine
  * basis and driver code associated with this device is no longer being updated.
- * Red Hat may fix security-related and critical issues.  Support for this device
+ * EuroLinux may fix security-related and critical issues.  Support for this device
  * will be disabled in a future major release and users deploying this device
  * should plan to replace the device in production systems.
  *
@@ -53,7 +53,7 @@
  * @driver_name: driver name
  *
  * Called to notify users that a driver will no longer be tested on a routine
- * basis and the driver code is no longer being updated.  Red Hat may fix
+ * basis and the driver code is no longer being updated.  EuroLinux may fix
  * security-related and critical issues.  Support for this driver will be
  * disabled in a future major release, and users should replace any affected
  * devices in production systems.
@@ -151,9 +151,9 @@
  *
  * Called to minimize the support status of a new driver.  This does TAINT the
  * kernel.  Calling this function indicates that the driver or subsystem has
- * had limited testing and is not marked for full support within this RHEL
- * minor release.  The next RHEL minor release may contain full support for
- * this driver.  Red Hat does not guarantee that bugs reported against this
+ * had limited testing and is not marked for full support within this EuroLinux
+ * minor release.  The next EuroLinux minor release may contain full support for
+ * this driver.  EuroLinux does not guarantee that bugs reported against this
  * driver or subsystem will be resolved.
  */
 void mark_tech_preview(const char *msg, struct module *mod)
