diff --color -urN linux-5.14.0-162.18.1.el9_1/arch/x86/boot/header.S linux-5.14.0-162.18.1.el9_1.NX/arch/x86/boot/header.S
--- linux-5.14.0-162.18.1.el9_1/arch/x86/boot/header.S	2023-02-09 09:40:45.000000000 +0100
+++ linux-5.14.0-162.18.1.el9_1.NX/arch/x86/boot/header.S	2023-03-15 13:12:50.134161685 +0100
@@ -163,7 +163,7 @@
 	.long	0x200				# SizeOfHeaders
 	.long	0				# CheckSum
 	.word	IMAGE_SUBSYSTEM_EFI_APPLICATION	# Subsystem (EFI application)
-	.word	0				# DllCharacteristics
+	.word	IMAGE_DLL_CHARACTERISTICS_NX_COMPAT				# DllCharacteristics
 #ifdef CONFIG_X86_32
 	.long	0				# SizeOfStackReserve
 	.long	0				# SizeOfStackCommit
