From 588bf8548c3139f7c8c37284641369e0672bf820 Mon Sep 17 00:00:00 2001
From: Your Name <you@example.com>
Date: Thu, 16 Mar 2023 11:40:56 +0100
Subject: [PATCH] nx-compat

---
 post-process-pe.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/post-process-pe.c b/post-process-pe.c
index de8f4a3..f39fddd 100644
--- a/post-process-pe.c
+++ b/post-process-pe.c
@@ -42,7 +42,7 @@ static int verbosity;
 		0;                                               \
 	})
 
-static bool set_nx_compat = false;
+static bool set_nx_compat = true;
 
 typedef uint8_t UINT8;
 typedef uint16_t UINT16;
-- 
2.31.1

